core/src/fetch_stage.rs

```diff
@@ -34,21 +34,30 @@ impl FetchStage {
         sockets: Vec<UdpSocket>,
         tpu_forwards_sockets: Vec<UdpSocket>,
         tpu_vote_sockets: Vec<UdpSocket>,
+        alpenglow_socket: UdpSocket,
         exit: Arc<AtomicBool>,
         poh_recorder: &Arc<RwLock<PohRecorder>>,
         coalesce: Option<Duration>,
-    ) -> (Self, PacketBatchReceiver, PacketBatchReceiver) {
+    ) -> (
+        Self,
+        PacketBatchReceiver,
+        PacketBatchReceiver,
+        PacketBatchReceiver,
+    ) {
         let (sender, receiver) = unbounded();
         let (vote_sender, vote_receiver) = unbounded();
+        let (alpenglow_sender, alpenglow_receiver) = unbounded();
```

Comment:
This channel should be bounded (it is fed by streamer => can grow out of control if not emptied in time)

Reply:
Changed.